README.md

```diff
@@ -1440,6 +1440,7 @@ Update Time, five active automations, webhooks.
    * [internxt.com](https://internxt.com) – Internxt Drive is a zero-knowledge file storage service based on absolute privacy and uncompromising security. Sign up and get 10 GB for free, forever!
    * [degoo.com](https://degoo.com/) – AI based cloud storage with free up to 20 GB, three devices, 5 GB referral bonus (90 days account inactivity).
    * [MConverter.eu](https://mconverter.eu/) – Convert files in bulk. Supports many file formats, including new ones like [AVIF](https://mconverter.eu/convert/to/avif/). Extract all image frames from videos. Free for up to ten 100MB-files per day, processed in batches of two.
+   * [GSheetConnector](https://www.gsheetconnector.com) - Google Sheet Connector for WordPress based Contact Forms, Page Builder Forms and eCommerce Store, It helps to save the form submissions and woocommerce order entries into the selected spreadsheet in a real-time! 
```

Comment:
your submission needs to tick all the boxes, if it does please tick them and update here with whats included in the SaaS

thanks!